test/test-resize-from-1-to-0.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+var cluster = require('cluster');
+var master = require('../lib/master');
+var tap = require('tap');
+var workerCount = require('./helpers').liveWorkers;
+
+cluster.setupMaster({
+  exec: 'test/workers/null.js'
```

Comment:
Its the original code, all the refactored tests use the null worker, some of the new tests don't.